css/src/components/breadcrumbs.scss

```diff
@@ -0,0 +1,30 @@
+$breadcrumb-font-size: $font-size-8 !default;
+$breadcrumb-spacing-horizontal: 0.25em !default;
+$breadcrumb-spacing-vertical: 0.375em !default;
+$breadcrumb-separator: '\00200B/' !default;
+
+.breadcrumbs {
+	font-size: $breadcrumb-font-size;
+	list-style: none;
+
+	.breadcrumb-item {
+		display: inline;
+
+		@include desktop {
+			display: inline-block;
+			padding-block: $breadcrumb-spacing-vertical;
+		}
```

Comment:
To me the vertical padding is external to this css component and would be something applied by a utility class as-needed.

Reply:
Added the vertical spacing for those cases when there are a lot of breadcrumbs and don't fit into one line, so when wraps - having some space would look nicer than without it:

![image](https://user-images.githubusercontent.com/57374379/144491748-a5f116ff-897a-41a3-99aa-406894ce780c.png)

Reply:
Makes sense! In that case it should be applied unconditionally since it's a function of the number/length of crumbs rather than viewport width which will introduce the wrapping.

Reply:
Updated. Kept `inline-block` + `padding` to keep some spacing all the time. `inline` + `padding` combo doesn't do what I need and it's not worth spending more time on this, especially knowing we're going to truncate it